src/Schema/Events/Tap.ts

```diff
@@ -177,6 +204,7 @@ export interface TappedEntityGroup {
   destination_screen_owner_type: ScreenOwnerType
   destination_screen_owner_id?: string
   destination_screen_owner_slug?: string
+  featured?: boolean
```

Comment:
Historically the key `featured` has been fraught in the codebase, we have many instances of this being used with completely different meanings, which becomes confusing over times. Is there a more specific name possible? What does a featured item in the group look like?

Reply:
Great point! Changing to `curation_boost`